packages/caver-kct/src/kctHelper.js

```diff
@@ -406,705 +406,1897 @@ const kip7ByteCode =
 
 // KIP-17 token contract source code
-// https://github.com/klaytn/klaytn-contracts/tree/master/contracts/token/KIP17/KIP17Token.sol
+// ./KIP17Token.sol
 // The ABI and bytecode below are built via the following command.
-// solc --abi --bin --allow-paths . klaytn-contracts/contracts/token/KIP17/KIP17Token.sol
+// solc --abi --bin --optimize --optimize-run 200 KIP17Token.sol
```

Comment:
Don't we need to specify which solc version is used for the build?

Reply:
resolved by [42b4d18](https://github.com/klaytn/caver-js/pull/711/commits/42b4d18eae558ee1cfc532258a27ee5391b66ce7)